src/components/InputGroup/index.tsx

```diff
@@ -12,7 +12,10 @@ type FlexType = 'shrink' | 'grow'[];
  * @param {Array} userSettings - `flexSettings` prop optionally set by user
  * @returns {Array} e.g. ['shrink', 'grow', 'shrink']
  */
-export const getFlexSettings = (childCount: number, userSettings: FlexType): FlexType => {
+export const getFlexSettings = (
+  childCount: number,
+  userSettings: FlexType[]
```

Comment:
if `flexSettings` below is optional, and we're passing it into `getFlexSettings` as `userSettings`... then I think `userSettings` should also be optional, which would remove the need to pass empty arrays into this function.

Reply:
agree